code/modules/mining/minebot.dm

```diff
@@ -126,7 +126,7 @@
 		if(health >= maxHealth * 0.75)
 			. += "<span class='warning'>It looks slightly dented.</span>"
 		else if(health >= maxHealth * 0.25)
-			. += "<span class='warning'>It looks moderately dented.</span>"
+			. += "<span class='warning'>It looks <b>moderately</b> dented.</span>"
```

Comment:
I don't see anything changing the health HUD here as it says in your changelog, did you miss something?

Reply:
Looks like I forgot to get that pushed, gonna fix it now.